dumpling/export/status.go

```diff
@@ -48,6 +48,25 @@ func (d *Dumper) runLogProgress(tctx *tcontext.Context) {
 	}
 }
 
+type Midparams struct {
+	TotalTables       float64
+	CompletedTables   float64
+	FinishedBytes     float64
+	FinishedRows      float64
+	EstimateTotalRows float64
+}
+
+func (d *Dumper) GetParameters() (midparams *Midparams) {
+	conf := d.conf
+	var mid *Midparams
```

Comment:
```suggestion
	mid := &Midparams{}
```

you code will assign `nil` to `mid`, but here we want `mid` is pointed to a valid address of variable.

By the way please add an unit test for this function. If there would be an unit test you can find above problem on your own.